src/align.rs

```diff
@@ -161,17 +163,17 @@ impl<T: Copy, C: AlignedByteChunk> AlignedVec<T, C> {
         self.inner.as_mut_ptr().cast()
     }
 
-    /// Extracts a slice containing the entire vector.
+    /// Extract a slice containing the entire vector.
     pub fn as_slice(&self) -> &[T] {
-        // Safety: The first `len` elements have been initialized to `T`s in
-        // `Self::resize_with`.
+        // Safety: The first `len` elements have been
+        // initialized to `T`s in `Self::resize_with`.
```

Comment:
Another minor nit that I'll mention here because I've seen it in a few of your PRs is that you tend to wrap comments a bit too aggressively. [The style guide](https://doc.rust-lang.org/nightly/style-guide/#comments) indicates that comments should be wrapped at 80 characters wide, unless they'd pass the 100 character line limit. So in this PR you've taken some comments that were fine at their previous width and wrapped them shorter. Like I said, not important and I wouldn't normally mention it, but I bring it up here since most of this PR is updating comments.

Reply:
Hmm, I didn't know the style guide said that.  100 is a bit more forgiving than just 80, so this one could fit all on one line.  I'll keep that in mind.  I do read code a lot on my phone, though, which is quite narrow.  In general, though, I prefer semantic line wrapping rather than any hard wrapping, since it reduces diffs and is IMO easier to read.

Reply:
What do you mean by "semantic line wrapping"?

Reply:
I mean wrapping lines at semantic breaks, like punctuation, end of phrases, other places you'd pause while reading it aloud, etc.  That way if you change one word in a line, it generally would only affect that line, as opposed to with hard wrapping, it might cascade down to all subsequent lines.